### ISSUE TITLE
Persistent state storage

### ISSUE THREAD

**Issue by pwarren** [MichaelRColton/PSDR#8]
It'd be nice to have the last used menu settings come back when the radio's turned on, with a way to reset it too I guess.

I'd imagine we'd need a menu state struct, which can be written to FLASH memory at some interval, maybe a second or so after a change?

**Comment by MichaelRColton** [comment]
Agreed. Depending on what power state we drop down into, we may be able to preserve those things in memory without too much of a hit to standby current.
